libraries/lib-import-export/LibsndfileTagger.h

```diff
@@ -10,20 +10,53 @@
 **********************************************************************/
 #pragma once
 
+#include "AcidizerTags.h"
+
 #include <memory>
 #include <string>
 
 typedef struct SNDFILE_tag SNDFILE;
 
-namespace LibFileFormats
-{
-struct AcidizerTags;
-}
-
 namespace LibImportExport
 {
 namespace Test
 {
+/*!
+ * @brief In the Audacity code, we only are interested in
+ * LibFileFormats::AcidizerTags and the information it carries, but for testing
+ * we need to mimic more than one way ACID metadata may be set out there. We
+ * therefore need to be able to set tempo, beats and one-shot status
+ * independently.
+ */
+struct AcidizerTags : LibFileFormats::AcidizerTags
+{
+   struct Beats
+   {
+      explicit Beats(int beats)
+          : beats { beats }
+      {
+      }
+      const int beats;
+   };
+
+   AcidizerTags(OneShot)
+       : LibFileFormats::AcidizerTags { OneShot {} }
+   {
+   }
+
+   AcidizerTags(Loop loop)
+       : LibFileFormats::AcidizerTags { loop }
+   {
+   }
```

Comment:
You might simplify away these two constructors with
`using LibFileFormats::AcidizerTags::AcidizerTags`